surah/migrations/0002_auto_20180908_0913.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.1 on 2018-09-08 09:13
```

Comment:
Why do we need this migration? Can't you just update the first file?
Secondly, is it a good practice to update generated Django files? I'm not completely sure, but maybe.

Reply:
Separating model creating and database seeding should be ok. Not auto created but with  makemigrations --empty yourappname